tests/cases/fourslash/moveToFile_overloads1.ts

```diff
@@ -14,7 +15,8 @@ verify.moveToFile({
     newFileContents: {
         "/a.ts": "",
         "/add.ts":
-`function add(x: number, y: number): number;
+`
```

Comment:
I mean this new line that exists in the target file. I added the `@newline` option to be sure that files use the same format. Does that make sense?

Reply:
The thing I was confused about is whether or not this relies on the new lines in the git checkout. Typically we use CRLF so I just didn't want things to fail in different ways. That's what I meant when I was referring to normalization; there are some fourslash calls which normalize things for this reason.

(But, I'm not quite at my desk so will give this a proper look in a bit)